frontend/src/metabase/static-viz/lib/numbers.ts

```diff
@@ -24,74 +25,9 @@ const DEFAULT_OPTIONS = {
 };
 
 export const formatNumber = (number: number, options?: NumberFormatOptions) => {
-  const {
-    number_style,
-    currency,
-    currency_style,
-    number_separators: [decimal_separator, grouping_separator],
-    decimals,
-    scale,
-    prefix,
-    suffix,
-    compact,
-  } = handleSmallNumberFormat(number, { ...DEFAULT_OPTIONS, ...options });
+  const { prefix, suffix } = { ...DEFAULT_OPTIONS, ...options };
 
-  function createFormat(compact?: boolean) {
-    if (compact) {
-      return new Intl.NumberFormat("en", {
-        style: number_style !== "scientific" ? number_style : "decimal",
-        notation: "compact",
-        compactDisplay: "short",
-        currency: currency,
-        currencyDisplay: currency_style,
-        useGrouping: true,
-        maximumFractionDigits: decimals != null ? decimals : 2,
-      });
-    }
-
-    return new Intl.NumberFormat("en", {
-      style: number_style !== "scientific" ? number_style : "decimal",
-      notation: number_style !== "scientific" ? "standard" : "scientific",
-      currency: currency,
-      currencyDisplay: currency_style,
-      useGrouping: true,
-      minimumFractionDigits: decimals,
-      maximumFractionDigits: decimals != null ? decimals : 2,
-    });
-  }
-
-  const format = createFormat(compact);
-
-  const separatorMap = {
-    ",": grouping_separator || "",
-    ".": decimal_separator,
-  };
-  const formattedNumber = format
-    .format(number * scale)
-    .replace(/,|\./g, separator => separatorMap[separator as "." | ","]);
-
-  return `${prefix}${formattedNumber}${suffix}`;
-};
-
-// Simple hack to handle small decimal numbers (0-1)
-function handleSmallNumberFormat<T>(value: number, options: T): T {
```

Comment:
Testing locally. Now percentage formatting works, so we don't need this fix anymore
![image](https://user-images.githubusercontent.com/1937582/203825463-309765d7-72ef-4ed0-b3b3-e551a16a994e.png)

This hack was introduced in https://github.com/metabase/metabase/pull/25361 as a means to remedy percentage formatting which we couldn't do at the time.